sympy/parsing/tests/test_latex.py

```diff
@@ -179,10 +193,13 @@ def test_import():
     ("\\sum_{k = 1}^{10} k^2", Sum(k**2, (k, 1, 10))),
     ("\\sum_{n = 0}^{\\infty} \\frac{1}{n!}",
      Sum(_Pow(_factorial(n), -1), (n, 0, oo))),
+    ("\\Sigma_{n = 0}^{\\infty} \\frac{1}{n!}",
+     Sum(_Pow(_factorial(n), -1), (n, 0, oo))),
     ("\\prod_{a = b}^{c} x", Product(x, (a, b, c))),
     ("\\prod_{a = b}^c x", Product(x, (a, b, c))),
     ("\\prod^{c}_{a = b} x", Product(x, (a, b, c))),
     ("\\prod^c_{a = b} x", Product(x, (a, b, c))),
+    ("\\Pi^c_{a = b} x", Product(x, (a, b, c))),
```

Comment:
There is some discussion in another PR about this. I'm not sure if we want to parse `\Pi` as a product or `\Sigma` as a sum. What affect does that have on an expression that uses `\Pi` as a symbol?

@eric-wieser @asmeurer

Reply:
I don't know the scope of the sympy latex parser, but my assumption would be that we only care about parsing best-practice latex, not playing guessing games for nonsense input.

With that frame of mind, `\Pi` should just be parsed as a symbol called `Pi`. I used that as a variable only yesterday in a latex document.

Reply:
I think using \Pi as a variable is inherently misleading.
![Screen Shot 2020-09-02 at 2 27 16 PM](https://user-images.githubusercontent.com/28682735/91961207-8390c880-ed28-11ea-88f3-90f1f007446a.png)
https://www.wikiwand.com/en/Multiplication#/Products_of_sequences

Reply:
but yeah we could remove that from this PR and add it once we resolve this issue

Reply:
to your credit @eric-wieser I toyed around with the idea of using using `\Gamma` to represent the gamma function in sympy. but yeah it wouldn't be worth messing with the symbolic usage.
what do you guys think about using `\operatorname{\Gamma}` and `\operatorname{\Sigma}` instead. 
allowing them to be used as a variable `\Gamma` and a function ☝️  if needed

Reply:
Wikipedia lists symbolic uses of Pi here: https://en.wikipedia.org/wiki/Pi_(letter)#Symbol

Reply:
I don't understand why anyone would write `\operatorname{\Sigma}` instead of `\sum`, while also wanting it to mean sum in sympy. The gamma case seems fine to me.

Reply:
Yeah `\operatorname{\Sigma}` is a bad example cause we already have `\sum`,
but the idea of using symbols as both functions and variables when used appropriately seems appealing . like the gamma case.

Reply:
The LaTeX parser is inherently heuristic. If there is an ambiguity, it should ideally try to pick the case that is more often used.  I think for \Pi and \Sigma, it should be possible to detect in the parser if it is being used to represent a product/sum or if they are symbols.

The fact is that a lot of people write bad LaTeX, and we should be able to parse it if we can. `\Pi^c_{a = b}` is bad because it does the wrong thing in display mode, but I guarantee people write it because they don't know about `\prod`. 

Regarding variable names that represent functions, I think we need to support overriding the namespace. Even something like `\pi` could be a variable name in some contexts, but there's no way to tell the parser that. The best way would be to use something like `parse_latex(..., {'pi': Symbol('pi')})`.

Reply:
>The fact is that a lot of people write bad LaTeX, and we should be able to parse it if we can. \Pi^c_{a = b} is bad because it does the wrong thing in display mode, but I guarantee people write it because they don't know about \prod.

I agree!

>Regarding variable names that represent functions, I think we need to support overriding the namespace. Even something like \pi could be a variable name in some contexts, but there's no way to tell the parser that. The best way would be to use something like parse_latex(..., {'pi': Symbol('pi')}).

yeah! we could have something like that, we could specify tokens and give an explicit directive to the parser to treat them differently

Reply:
One of the reasons I'm excited about moving to Lark is it should make it a lot easier to allow users to extend the parser themselves. I'm not sure if that's even possible with the current antlr parser, and if it is, it wouldn't be easy.